vita3k/gui/src/settings_dialog.cpp

```diff
@@ -359,9 +359,9 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::PopStyleColor();
         ImGui::Spacing();
         if (!gui.modules.empty()) {
-            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Modules Mode");
+            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Modules mode");
             ImGui::Spacing();
-            ImGui::Checkbox("Experimental: LLE DriverUser", &config.lle_driver_user);
+            ImGui::Checkbox("LLE DriverUser", &config.lle_driver_user);
```

Comment:
fail, delete this change

Reply:
Not necessary to delete this change, its saying its experimental in the tooltip, no need to clutter the screen

Reply:
is needed, some user aleady enable it and after complain :/